app/views/site/contribute.erb

```diff
@@ -0,0 +1,130 @@
+<div class="container">
+  <section class="contributing-intro jumbotron">
+    <h1>Contributing Guide</h1>
+
+    <p class="lead">
+      <strong>So you want to contribute to exercism?</strong><br />
+      That is great news! We welcome all sort of contributions and will do everything we can to help you get started.
+    </p>
+
+    <p>
+      <strong>Exercism is a project comprised of several repositories and these can be divided basically in:</strong>
+
+      <ol>
+        <li><strong>The website</strong>: Its here that we can improve exercism features or layout.</li>
+        <li><strong>Command line interface</strong>: Fetch exercises and submit solutions.</li>
+        <li><strong>Language track repositories</strong>: Each language has its own repository with available problems.</li>
```

Comment:
Maybe:

_Contain the language-specific implementations of exercises. Each language has its own repository._

(Can you tell I'm trying to normalize the format of the sentences? I'm not quite there yet.)